src/bar-chart.js

```diff
@@ -73,7 +73,9 @@ class BarChart extends AbstractChart {
       horizontalLabelRotation = 0,
       withInnerLines = true,
       showBarTops = true,
-      segments = 4
+      segments = 4,
+      formatYLabel = yLabel => yLabel,
```

Comment:
There are 2 ways to pass configuration in this lib, some things are done through component props, other things are done using `chartConfig`, which is itself a prop that all components in this lib take. `chartConfig` is sort of a common configuration component for all charts and graphs. This is why label formatting functions are passed through this prop, because they are common across components. What you seem to be doing here is moving these formatting functions from `chartConfig` prop into their own props.

I'm closing this PR because this is a *breaking change* for no good reason.

Reply:
Thank you for to have taken the time of this explanation!